src/Generating/EnderDragonFightStructuresGen.cpp

```diff
@@ -57,7 +58,7 @@ const std::array<Vector3i, 26> cEnderDragonFightStructuresGen::m_CageAir =
 
 
 
-cEnderDragonFightStructuresGen::cEnderDragonFightStructuresGen(int a_Seed, const AString &a_TowerProperties, int a_Radius) :
+cEnderDragonFightStructuresGen::cEnderDragonFightStructuresGen(int a_Seed, const AString &a_TowerProperties, int a_Radius, int a_ChunkWidth) :
```

Comment:
Why add this as a parameter when you can just access it from cChunkDef?

Reply:
It failed on linking with MVSC with the error that the symbol didn't exist. See my earlier comment

Reply:
Whoops, gotta learn how to read I guess ;)

Would moving everything to some kind of `Initialize` method fix the problem?

Reply:
Gonna try that later 👍🏻

Reply:
Now it's getting weird... Still doesn't compiler...